Configuration/Eras/python/Modifier_run3_directSim_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+run3_directSim =  cms.Modifier()
```

Comment:
run3_directSim might not be a nice name. We should have the same behaviour also in Run 2 (or anyway make it clear that Run 2 works in the same way).

Reply:
Hmm, that might be a good suggestion, if I understand correctly I can just name it `run_direct_sim` and it will be setting `useNewLHCInfo` to `false`

Reply:
I would also add "pps" to the name of the modifier

Reply:
> I would also add "pps" to the name of the modifier

Using `ctpps` name would be more consistent with the existing modifiers.

Reply:
Yup, I believe that something like `ctpps_directSim` could be more appropriate, as we don't really distinguish between Run 2 and Run 3 for what concerns the usage of LHCInfo in the PPS direct simulation

Reply:
Changed in 861740283a6fa83d253f35682792539d48aa6c26